qa/L0_infer/test.sh

```diff
@@ -39,6 +39,7 @@ export CUDA_VISIBLE_DEVICES=0
 
 CLIENT_LOG_BASE="./client"
 INFER_TEST=infer_test.py
+EXPECTED_NUM_TESTS="42"
```

Comment:
This might be different for the shared memory variants of L0_infer. You can run the system shared memory variant by setting TEST_SYSTEM_SHARED_MEMORY=1 and confirm the same. Specifically the tests that use class output instead of raw are not run for the shared memory. We can discuss this offline if you have any questions.